experiment/simulation/css/main.css

```diff
@@ -1 +1,457 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+
+/* disabling text selection */
+body {
+  -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+     -khtml-user-select: none; /* Konqueror HTML */
+       -moz-user-select: none; /* Old versions of Firefox */
+        -ms-user-select: none; /* Internet Explorer/Edge */
+            user-select: none; /* Non-prefixed version, currently
+                                  supported by Chrome, Edge, Opera and Firefox */
+}
+
+text {
+  pointer-events:none;
+}
+
+.drag-drop {
+  display: inline-block;
+  position: absolute;
+  top: 0px;
+  width: 50px;
+  height: 50px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  z-index: 1;
+  background-size: 100%;
+}
+
+.rsflipflop {
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 150px;
+  width: 100px;
+  background-image: url("../images/ff_sr.svg");
+  background-size: 130%;
+}
+
+.jkflipflop {
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 150px;
+  width: 100px;
+  background-image: url("../images/ff_jk.svg");
+  background-size: 130%;
+}
+
+.dflipflop {
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 150px;
+  width: 100px;
+  background-image: url("../images/ff_d.svg");
+  background-size: 130%;
+}
+
+.logic-gate{
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.and {
+  background-image: url("../images/and.svg");
+}
+
+.not {
+  background-image: url("../images/not.svg");
+}
+
+.or {
+  background-image: url("../images/or.svg");
+}
+
+.xor {
+  background-image: url("../images/xor.svg");
+}
+
+.xnor {
+  background-image: url("../images/xnor.svg");
+}
+
+.nand {
+  background-image: url("../images/nand.svg");
+}
+
+.nor {
+  background-image: url("../images/nor.svg");
+}
+
+.threeipnand {
```

Comment:
please put hyphens to make it readable
three-ip-nand

Reply:
done